src/k2/tso/client/Client.cpp

```diff
@@ -71,7 +71,20 @@ seastar::future<> TSOClient::_doGetTSOEndpoints(dto::GetTSOEndpointsRequest &req
         }
         _tsoErrorBound = resp.minTransTime;
         _tsoServerEndpoint = RPC().getTXEndpoint(resp.endpoints[0]);
```

Comment:
_tsoServerEndpoint isn't necessary anymore is it?

Reply:
Removed.